utils/exporters/blender/addons/io_three/__init__.py

```diff
@@ -522,6 +676,14 @@ class ExportThree(bpy.types.Operator, ExportHelper):
         default=2)
 
     def invoke(self, context, event):
+        settings = context.scene.get(constants.EXPORT_SETTINGS_KEY)
+        if settings:
+            try:
+                restore_export_settings(self.properties, settings)
+            except:
```

Comment:
Blank exception handling! Yikes! This is a big pet peeve of mine as this alone is responsible for many late nights and weekends trying to fix broken animation pipelines because the problem was masked.

At the very least, if you aren't going to name the exception(s), is to avoid letting it fail silently by printing a warning and the details of the traceback before continuing. Example: https://github.com/mrdoob/three.js/blob/dev/utils/exporters/blender/addons/io_three/exporter/__init__.py